src/core/ts/background-scripts/background.js/telemetry/metrics.yaml

```diff
@@ -46,6 +46,22 @@ metadata:
     notification_emails:
       - anatal-all@mozilla.com
     expires: 2021-08-10
+  firefox_client_id:
+    type: string
+    lifetime: ping
+    send_in_pings:
+      - custom
+    description: |
+      Firefox Telemetry client id.
+    bugs:
+      - https://github.com/mozilla-extensions/bergamot-browser-extension/issues/92
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1694813
+    data_sensitivity:
+      - interaction
```

Comment:
It allows us to join also with interaction level Firefox telemetry so I argue that it has the privacy impact of interaction level.

Reply:
The metric itself is purely technical and doesn't tell about any interactions. It might allow joining to any data in telemetry storage including level 4 from some random projects as soon as they use the same telemetry id. It doesn't make it level 4. Our intention is to obtain another group of technical level metrics with this one - user environment. I suggest to fix. It doesn't really matter though.